server/src/test/scala/drt/server/feeds/lhr/LHRForecastFeedSpec.scala

```diff
@@ -0,0 +1,32 @@
+package drt.server.feeds.lhr
+
+import akka.actor.{Actor, Props}
+import akka.util.Timeout
+import server.feeds.ArrivalsFeedFailure
+import services.crunch.CrunchTestLike
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class SlowResponseActor extends Actor {
+  override def receive: Receive = {
+    case _ =>
+      Thread.sleep(1000L)
```

Comment:
should we make the wait periods lower rather than adding a second to our test suite?

Reply:
oh sorry, I see it won't actually hit this timeout.